src/Whitelist.php

```diff
@@ -244,6 +260,16 @@ public function getRecordedWhitelistedClasses(): array
         return array_values($this->whitelistedClasses);
     }
 
+    public function getNamespacesWhitelistIsInverted(): bool
+    {
+        return $this->namespacesWhitelistIsInverted;
+    }
+
+    public function setNamespacesWhitelistIsInverted(bool $value): void
```

Comment:
is the setter necessary?

Reply:
It is, since I didn't include this in the class constructor (the existing 8 arguments seemed too much already to add yet another one, and modifying unit tests would be a pain).